src/packages/@ncigdc/modern_components/RepoCasesTable/utils.js

```diff
@@ -0,0 +1,94 @@
+// @flow
```

Comment:
Is there any difference between this and the one in `tableModels/utils.js`?

Reply:
ok removing